docs/src/main/markdown/customizations.md

```diff
@@ -721,3 +721,10 @@ message BarMessage {
 }
 ```
 
+In ScalaPB 0.11.4, you can also add annotations to the enum values and the `Unrecognized` case class:
+TODO:mapek
```

Comment:
Can you fix the example? Once merged this gets automatically deployed to the live site.

Reply:
sure